src/diffpy/morph/morphs/morphfuncy.py

```diff
@@ -1,9 +1,61 @@
+"""class MorphFuncy -- apply a user-supplied python function to the y-axis."""
+
 from diffpy.morph.morphs.morph import LABEL_GR, LABEL_RA, Morph
 
 
 class MorphFuncy(Morph):
-    """Apply the user-supplied Python function to the y-coordinates of the
-    morph data"""
+    """General morph function that applies a user-supplied function to the
```

Comment:
This needs one high level statement of <=72 characters, then blank line, then a longer description